install.py

```diff
@@ -89,6 +90,25 @@ def check_desktop_env():
         return VERSION_GUSSERS.get(desktop_environment, lambda: None)()
     return None
 
+def check_thunar():
+    thunar = os.path.isfile('/etc/xdg/tumbler/tumbler.rc')
+    if thunar:
+        f=open("/etc/xdg/tumbler/tumbler.rc", "a+")
```

Comment:
can you use a managed context for this instead?

Something like (note the preference for longer variable names and for single instead of double quotes):

```
with(open('/etc/xdg/tumbler/tumbler.rc')) as tumbler_config:
    content = mmap.mmap(tumbler_config.fileno(), 0, access=mmap.ACCESS_READ)
    if content.find('# Ebook thumbnailer') == -1:
        tumbler_config.write('\n')
        ...
        print('Registered thumbnailer in tumbler.rc')
    else:
        print('Thumbnailer already present in tumbler.rc')
```